### DISCUSSION TITLE
Error on package:discover

### DISCUSSION THREAD

**Discussion by TwilightDuck** [spatie/laravel-feed#162] [Bugs]
This occured after a composer update. I do not have the configuration file published.

```` @php artisan package:discover

   ErrorException 

  Undefined array key "format"

  at vendor/spatie/laravel-feed/src/Helpers/ConfigurationValidator.php:15
     11▕     {
     12▕         $feeds = $feeds ?? (array)config('feed.feeds', []);
     13▕ 
     14▕         foreach ($feeds as $name => $config) {
  ➜  15▕             if (! in_array($config['format'], ['atom', 'json', 'rss'])) {
     16▕                 throw InvalidConfiguration::unrecognizedFormat($name, $config['format']);
     17▕             }
     18▕ 
     19▕             if (! View::exists($config['view'] ?? 'feed::atom')) {

      +10 vendor frames 
  11  [internal]:0
      Illuminate\Foundation\Application::Illuminate\Foundation\{closure}()

      +5 vendor frames 
  17  artisan:37
      Illuminate\Foundation\Console\Kernel::handle()
Script @php artisan package:discover handling the post-autoload-dump event returned with error code 1
````

**Comment by PHLAK** [comment]
I am also experiencing this issue in one of my environments (but not locally). I have the `format`  option set to `atom` but it's still giving me this error.

**Comment by anhofmann** [comment]
I have the same problem, but only on production, not locally. I have no config at all, I wasn't even aware of having this package in my dependency tree. Furthermore, I found out that this package require `laravel-feed`: `spatie/laravel-mailcoach`.

PHP on production:
```
PHP 8.0.7 (cli) (built: Jun 14 2021 15:20:56) ( NTS )
Copyright (c) The PHP Group
Zend Engine v4.0.7, Copyright (c) Zend Technologies
    with Zend OPcache v8.0.7, Copyright (c), by Zend Technologies
```

PHP local:
```
PHP 8.0.7 (cli) (built: Jun  4 2021 21:25:51) ( NTS )
Copyright (c) The PHP Group
Zend Engine v4.0.7, Copyright (c) Zend Technologies
    with Zend OPcache v8.0.7, Copyright (c), by Zend Technologies
```

**Comment by christopherarter** [comment]
I'm also experiencing this exact issue on upgrading Mailcoach from v3 to v4. I have published the configuration file, but still see the error. Any ideas on this @freekmurze ?

**Comment by geowrgetudor** [comment]
If you'd check `vendor/spatie/laravel-feed/src/Helpers/ConfigurationValidator.php:15` you will notice that it checks for the `format` key in each feed you have defined. So make sure you have it added to each custom feed inside `config/feeds.php`. This wasn't mandatory before btw.

**Reply by frknasir** [reply]
This is it! 

It usually happens after upgrade. Thanks!

**Reply by matheusramalhosilva** [reply]
How do you do that, could you show?
